eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/MetaHandler.java

```diff
@@ -120,29 +119,34 @@ void get(HttpExchange httpExchange) throws IOException {
             httpExchange.sendResponseHeaders(200, Objects.requireNonNull(result).getBytes(Constants.DEFAULT_CHARSET).length);
             out.write(result.getBytes(Constants.DEFAULT_CHARSET));
         } catch (NullPointerException e) {
-            // registry not initialized, return empty list
-            String result = JsonUtils.toJSONString(new ArrayList<>());
-            httpExchange.sendResponseHeaders(200, Objects.requireNonNull(result).getBytes(Constants.DEFAULT_CHARSET).length);
-            out.write(result.getBytes(Constants.DEFAULT_CHARSET));
+            try (OutputStream out = httpExchange.getResponseBody()) {
+                // registry not initialized, return empty list
+                String result = JsonUtils.toJSONString(new ArrayList<>());
```

Comment:
The L104, L122 and L135 should be using the same `OutputStream` object.

Reply:
Done